census/loaders/index.js

```diff
@@ -210,16 +216,12 @@ var loadData = function(options, models) {
     .spread((data, site) => {
       return Promise.all(_.map(data, dataObj => {
         // Allow custom data mapping
-        let createData = _.chain(
-          _.isFunction(options.mapper) ? options.mapper(dataObj, site) : dataObj
-        )
+        let createData = _.isFunction(options.mapper) ?
+          options.mapper(dataObj, site) : dataObj;
```

Comment:
That's a weird indentation, IMHO it would be better like this, if you really want to break it down to more lines:

```javascript
let variable = someFunction(param1, param2)
    ? valueIfYes
    : valueIfNo;
```
but in this case I think it could also live on a single line.